### ISSUE TITLE
Tables not supported

### ISSUE THREAD

**Issue by cweiske** [bambax/markitdown.medusis.com#4]
I pasted a table, but no markdown table markup was generated

**Comment by bambax** [comment]
Indeed, tables are currently not supported. The reason is, tables are not part of original Markdown and I wasn't sure at the time which idiom to choose. What would you suggest in that regard: what do you expect

    <table><th><td></td></th><tr><td></td></tr></table>

to transform to?

**Comment by cweiske** [comment]
I just see that Commonmark does not specify tables at all.

Github's (and gitlab's) Markdown support tables:
- https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown
- http://docs.gitlab.com/ee/user/markdown.html#tables

```
| header 1 | header 2 |
| -------- | -------- |
| cell 1   | cell 2   |
| cell 3   | cell 4   |
```
